src/main/kotlin/com/avail/descriptor/representation/AbstractDescriptor.kt

```diff
@@ -0,0 +1,4202 @@
+	 * @param argumentPhraseTuple
+	 * @return A_Tuple
+	 */
+	abstract fun o_LookupMacroByPhraseTuple (
+		self: AvailObject,
+		argumentPhraseTuple: A_Tuple): A_Tuple
+
+	abstract fun o_ExpressionAt (self: AvailObject, index: Int): A_Phrase
+
+	abstract fun o_ExpressionsSize (self: AvailObject): Int
+
+	abstract fun o_ParsingPc (self: AvailObject): Int
+
+	abstract fun o_IsMacroSubstitutionNode (self: AvailObject): Boolean
+
+	abstract fun o_LastExpression (self: AvailObject): A_Phrase
+
+	abstract fun o_MessageSplitter (self: AvailObject): MessageSplitter
+
+	abstract fun o_StatementsDo (
+		self: AvailObject,
+		continuation: (A_Phrase) -> Unit)
+
+	abstract fun o_MacroOriginalSendNode (self: AvailObject): A_Phrase
+
+	abstract fun o_EqualsInt (self: AvailObject, theInt: Int): Boolean
+
+	abstract fun o_Tokens (self: AvailObject): A_Tuple
+
+	abstract fun o_ChooseBundle (
+		self: AvailObject,
+		currentModule: A_Module): A_Bundle
+
+	abstract fun o_ValueWasStablyComputed (self: AvailObject): Boolean
+
+	abstract fun o_ValueWasStablyComputed (
+		self: AvailObject,
+		wasStablyComputed: Boolean)
+
+	abstract fun o_Definition (self: AvailObject): A_Definition
+
+	abstract fun o_NameHighlightingPc (self: AvailObject): String
+
+	abstract fun o_SetIntersects (self: AvailObject, otherSet: A_Set): Boolean
+
+	abstract fun o_RemovePlanForDefinition (
+		self: AvailObject,
+		definition: A_Definition)
+
+	abstract fun o_DefinitionParsingPlans (self: AvailObject): A_Map
+
+	abstract fun o_EqualsListNodeType (
+		self: AvailObject,
+		aListNodeType: A_Type): Boolean
+
+	abstract fun o_SubexpressionsTupleType (self: AvailObject): A_Type
+
+	abstract fun o_IsSupertypeOfListNodeType (
+		self: AvailObject,
+		aListNodeType: A_Type): Boolean
+
+	abstract fun o_UniqueId (self: AvailObject): Long
+
+	abstract fun o_LazyTypeFilterTreePojo (self: AvailObject): A_BasicObject
+
+	abstract fun o_AddPlanInProgress (
+		self: AvailObject,
+		planInProgress: A_ParsingPlanInProgress)
+
+	abstract fun o_ParsingSignature (self: AvailObject): A_Type
+
+	abstract fun o_RemovePlanInProgress (
+		self: AvailObject,
+		planInProgress: A_ParsingPlanInProgress)
+
+	abstract fun o_ModuleSemanticRestrictions (self: AvailObject): A_Set
+
+	abstract fun o_ModuleGrammaticalRestrictions (self: AvailObject): A_Set
+
+	abstract fun o_FieldAt (self: AvailObject, field: A_Atom): AvailObject
+
+	abstract fun o_FieldAtPuttingCanDestroy (
+		self: AvailObject,
+		field: A_Atom,
+		value: A_BasicObject,
+		canDestroy: Boolean): A_BasicObject
+
+	abstract fun o_FieldTypeAt (self: AvailObject, field: A_Atom): A_Type
+
+	abstract fun o_ParsingPlan (self: AvailObject): A_DefinitionParsingPlan
+
+	abstract fun o_CompareFromToWithIntTupleStartingAt (
+		self: AvailObject,
+		startIndex1: Int,
+		endIndex1: Int,
+		anIntTuple: A_Tuple,
+		startIndex2: Int): Boolean
+
+	abstract fun o_IsIntTuple (self: AvailObject): Boolean
+
+	abstract fun o_LexerMethod (self: AvailObject): A_Method
+
+	/**
```

Comment:
Remove remaining comments of this form.

Reply:
Done. I searched the whole code base and cleaned up anything like this; there were a few more in Stacks.